code/controllers/evacuation/evacuation_pods.dm

```diff
@@ -28,6 +28,10 @@
 			if (pod.arming_controller)
 				pod.arming_controller.arm()
 
+			if (istype(pod.active_docking_controller, /datum/computer/file/embedded_program/docking/simple/escape_pod))
```

Comment:
this is a hack, not the way to do things

The pods need a bit of refactoring because I reworked shuttle code and didn't get around to them